wafer_space/projects/admin.py

```
@@ -260,3 +260,200 @@ def git_commit_sha_short(self, obj: PrecheckImageRevision) -> str:
 
 # Import compliance admin to register it
 from .admin_compliance import ProjectComplianceCertificationAdmin  # noqa: E402, F401
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `E402`, `F401`)

Remove unused `noqa` directive

(RUF100)